easybuild/tools/module_naming_scheme/lowercase_module_naming_scheme.py

```diff
@@ -0,0 +1,33 @@
+import os
+import re
+
+from easybuild.tools.module_naming_scheme.mns import ModuleNamingScheme
+from easybuild.tools.module_naming_scheme.utilities import det_full_ec_version
+
+
+class LowercaseModuleNamingScheme(ModuleNamingScheme):
+    """Class implementing a lowercase module naming scheme."""
+
+    REQUIRED_KEYS = ['name', 'version', 'versionsuffix', 'toolchain']
+
+    def det_full_module_name(self, ec):
+        """
+        Determine full module name from given easyconfig, according to the EasyBuild module naming scheme.
+        :param ec: dict-like object with easyconfig parameter values (e.g. 'name', 'version', etc.)
+        :return: string with full module name <name>/<installversion>, e.g.: 'gzip/1.5-goolf-1.4.10'
+        """
+        return os.path.join(ec['name'], det_full_ec_version(ec)).lower()
```

Comment:
This could be simplified to

```python
return super(EB_LowercaseEasyBuildMNS, self).det_full_module_name().lower()
```

Reply:
Thanks for the hint, however if I change it as you suggested, I keep getting the `NameError` below: 
```
NameError: global name 'EB_LowercaseEasyBuildMNS' is not defined
```
Maybe I'm missing something trivial...

Reply:
@lucamar that's because you did not yet implement the suggested change by @boegel to `LowercaseEasyBuildMNS` (see above).